README.md

```diff
@@ -10,11 +10,15 @@ Supports the latest CSS syntax and features, including custom properties, range
 $ npm install stylelint
 ```
 
-_Note: see issue [#133](https://github.com/stylelint/stylelint/issues/133) for details of our progress towards more accurate line numbers_.
+_The reported line numbers of some warnings may be inaccurate. See issue [#133](https://github.com/stylelint/stylelint/issues/133) for details of our progress towards resolving this._
 
 ## Usage
 
-For now you must use the linter as a [PostCSS](https://github.com/postcss/postcss#usage) plugin directly. You can either use a [postcss runner](https://github.com/postcss/postcss#usage), like `grunt`, `gulp` or `webpack`. For example, using `gulp`:
+You must, for now, use the linter as a [PostCSS plugin](https://github.com/postcss/postcss#usage). This means you can either use a PostCSS runner, such as: [`gulp-postcss`](https://github.com/postcss/gulp-postcss), [`grunt-postcss`](https://github.com/nDmitry/grunt-postcss) and [`postcss-loader`](https://github.com/postcss/postcss-loader), or you can use the PostCSS JS API.
```

Comment:
My former-English-teacher punctuation fixes:
"You can either use a PostCSS runner -- such as [`gulp-postcss`](https://github.com/postcss/gulp-postcss), [`grunt-postcss`](https://github.com/nDmitry/grunt-postcss) and [`postcss-loader`](https://github.com/postcss/postcss-loader) -- or you can use the PostCSS JS API directly."